vpn/src/main/java/com/duckduckgo/mobile/android/vpn/processor/tcp/TcpPacketProcessor.kt

```diff
@@ -231,7 +231,6 @@ constructor(
             queues.networkToDevice.offerFirst(buffer)
 
             try {
-                selectionKey.cancel()
```

Comment:
Closing the channel cancels the keys, so we don't need to keep track of the `selectionKey` at all.